todo_app/app.py

```diff
@@ -4,23 +4,23 @@
 from todo_app.flask_config import Config
 from todo_app.data.view_model import ViewModel
 
-def create_app():
-    app = Flask(__name__)
-    app.config.from_object(Config())
+#def create_app():
+app = Flask(__name__)
+app.config.from_object(Config())
```

Comment:
I think this will break your tests, make sure you get them running again before you merge, we'll need them in Module 7.

You can tell gunicorn to call create_app with `gunicorn 'todo_app.app:create_app()'`.